RallyHereDebugTool/Source/Private/RH_ImGuiUtilities.cpp

```diff
@@ -166,6 +166,162 @@ void ImGuiDisplayCustomData(const TMap<FString, FString>& CustomData, const FStr
 	}
 }
 
+void ImGuiDisplayModelData(const FRHAPI_Model* Model, const UStruct* Struct)
+{
+	TFunction<void(const FString& Key, FProperty const*, FProperty const*, uint8 const*)> DisplayValueForProperty;
```

Comment:
Rather than fully declaring the TFunction<>, this is a great spot to just use auto on the following line
That is...
auto DisplayValueForProperty = [&](...)